recipes/onnx2torch/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "onnx2torch" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f668f6ecb2f34029d4c85a66be2f4a329a3d5d2ed8513395e508322d20dcb859
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - pip
+    - python >=3.6
+    - torchvision >=0.9.0
+  run:
+    - numpy >=1.16.4
+    - onnx >=1.9.0
+    - python >=3.6
+    - torchvision >=0.9.0
```

There is a direct dependency on `pytorch >=1.8.0`.

```suggestion
    - pytorch >=1.8.0
    - torchvision >=0.9.0
```

PS: next time try to create the recipe with grayskull. It makes a recipe that is almost "merge-ready."

---

Okay. It was created by `conda skeleton pypi onnx2torch`.

```python
   - pytorch >=1.8.0
   - torchvision >=0.9.0
```
It didn't work on osx build test due to inconsistent versions of pytorch and torchvision, but ok, i revert it.

---

Works now, thanks!